public/app/types/alerting.ts

```diff
@@ -142,36 +142,37 @@ export interface NotificationChannelState {
   notificationChannel: any;
 }
 
-export interface IntegrationError {
-  lastError: string;
+export interface IntegrationStatus {
+  lastError?: null | string;
```

Comment:
Is `null` necessary if the type is `lastError?`?

Reply:
Actually I added this null | because I remember someone said that this could be null at some point, but, in any case this is in progress and probably will be updated when we have the stable conract of the api response

Reply:
I added the  optional `?` because in the receivers.json example there are some cases where this is not defined.

Reply:
Ok, we can adjust the types when the final API is in place

Reply:
exaclty